### PR TITLE
174565856 start the quiz

### PR DESCRIPTION
**Pull request by NickCallaghan**

### CODE REVIEW

**Comment by NickCallaghan** [comment]
What type of PR is this? (check all applicable)
☐ :recycle: Refactor
☑ :sparkles: Feature
☐ :bug: Bug Fix
☐:construction_worker: Optimization
☐ :memo: Documentation Update
☐ :bookmark: Release
☐ :triangular_flag_on_post: Other

**Description** 
- Added react router
- Added constants for the routes /, /start and /quiz
- Created Quiz Welcome component mounted to /start
- Created a quiz holding component as a holding page

**Related Tickets & Documents**
[Start the Quiz](https://www.pivotaltracker.com/story/show/174565856)


**Added tests?**
☑ :+1: yes
☐ :no_good: no, because they aren't needed
☐ :raising_hand: no, because I need help

**Added to documentation?**
☐ :scroll: readme
☑ :no_good: no documentation needed

**Screenshots** 
Screenshot 2020-08-31 at 11 56 58
Screenshot 2020-08-31 at 11 59 29

**Comment by NickCallaghan** [comment]
User Flow and Urls (1)

**Comment by NickCallaghan** [comment]
I've added all the unit tests I can. I've not got any unit tests around the router component, not sure how. Wondering if it's actually even necessary, could be picked up on the e2e tests

**Comment by thiyagarajan-angappan** [comment]
Great work on the PR documentation and the before and after !

File: src/components/Text.tsx [hunk 1]
```diff
@@ -7,6 +7,7 @@ type TextProps = {
   fontStyle?: string;
   fontWeight?: number;
   textAlign: 'center';
+  padding?: string;
 };
 
 const Text: FC<TextProps> = ({
```
[the remark this answers is not in the document]
Comment (NickCallaghan) [reply src/components/Text.tsx:13]: Yeah for sure!

**Pull request merged**